main/createUiDefinition.json

```
@@ -186,6 +198,7 @@
     ],
     "outputs": {
       "jointoken": "[basics('jointoken')]",
+      "accessproxy": "[basics('accessproxy')]",
```

```suggestion
      "accessproxy": "[basics('httpproxy')]",
```
Except here everywhere name is `httpproxy`, use `accesproxy` only inside cloud-init user data

---

done